content/docs/command-reference/root.md

```diff
@@ -29,22 +30,78 @@ refer to a file in another directory.
 
 ```dvc
 $ dvc root
-
 .
 
 $ cd subdir
 $ dvc root
-
 ..
 ```
 
 ## Example: Referencing files
 
+Simplify file references when working in a subdirectory of a <abbr>DVC
+project</abbr>.
+
 ```dvc
 $ dvc root
-
 ../../../
 
 $ dvc run -d $(dvc root)/data/file.cvs ... \
     python $(dvc root)/scripts/something.py
 ```
+
+## Example: Output references
+
+Simplify output file or directory references.
+
+```dvc
+$ dvc root
+../../../
+
+$ dvc get -o $(dvc root)/root-model.pkl \
+    https://github.com/iterative/example-get-started model.pkl
+```
+
+## Example: Other commands
+
+Simplify other commands when working in a <abbr>DVC project</abbr>.
+
+```dvc
+$ dvc root
+../..
+
+$ tree $(dvc root)/data/
+../../data/
+├── data.xml
+├── data.xml.dvc
+...
+└── prepared
+    ├── test.tsv
+    └── train.tsv
+```
+
+## Example: Build reusable paths
+
+Build reusable paths to dependencies, scripts, or <abbr>data artifacts</abbr>
+from separate stages and subdirectories.
+
+```dvc
+$ cd more_stages/
+$ dvc run -n process_data \
+          -d data.in \
+          -d $(dvc root)/process_data.py \
```

Comment:
OK... I think what we need is a shell script that generates a dvc.yaml file instead of another `dvc run` example. Not required for this PR, so if you're not so familiar with Linux shell just leave it out for now please. Thanks

Reply:
I removed this for now. 

I looked into some related issues (#615) and [examples of scripts](https://github.com/bertbesser/dvc-walkthrough/blob/master/scripts/walkthrough.sh). 

These make sense within the scripting use case but instead of adding a (complicated) example that builds out a full set of stages, it would help to have the team vet a smaller example that fits the recommended usage.